lib/cartopy/tests/crs/test_transform_point.py

```diff
@@ -0,0 +1,17 @@
+# Copyright Cartopy Contributors
+#
+# This file is part of Cartopy and is released under the LGPL license.
+# See COPYING and COPYING.LESSER in the root of the repository for full
+# licensing details.
+import warnings
+
+import cartopy.crs as ccrs
+
+
+def test_transform_point():
```

Comment:
```suggestion
def test_transform_point_no_warning():
# Make sure we aren't warning on single-point numpy arrays
```
Can you move this over to `tests/test_crs.py` which already has a bunch of point transformation tests?

Reply:
sure thing